app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -1,10 +1,16 @@
 module PhysicalServerHelper::TextualSummary
   def textual_group_properties
-    %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    TextualGroup.new(
+      _("Properties"),
+      %i(name model product_name manufacturer machine_type serial_number ems_ref)
+    )
   end
 
   def textual_group_relationships
-    %i(host)
+    TextualGroup.new(
+      _("Relationships"),
+      %i(host)
```

Comment:
@GabrielSVinha Can you tell me more about the `host` relationship?

When I click on `host`, it takes me to the `host/show_list` page (with `Invalid input` flash message) which does not seem right.
<img width="1351" alt="screen shot 2017-05-08 at 2 58 29 pm" src="https://cloud.githubusercontent.com/assets/1538216/25827226/f745d176-33fe-11e7-943a-c4adc4920c8e.png">